Description: fix curses include for tests
 Tests did not work due to a missing include for ncurses
Author: xaizek <xaizek@lavabit.com>
Origin: upstream
Forwarded: no
Last-Update: 2012-10-26

--- a/tests/Makefile.inc
+++ b/tests/Makefile.inc
@@ -32,7 +32,7 @@
 LOCAL_OBJECTS := $(LOCAL_SOURCES:.c=.o)
 LOCAL_OBJECTS := $(addprefix bin/, $(notdir $(LOCAL_OBJECTS)))
 
-CFLAGS := -MD -g -Werror -I../seatest -DTEST -include ../../config.h
+CFLAGS := -MD -g -Werror -I../seatest -DTEST -include ../../config.h -I/usr/include/ncursesw
 LDFLAGS := -lpthread
 ifneq ($(OS),Windows_NT)
     LDFLAGS += `grep LIBS\ = ../../src/Makefile | sed 's/^[^=]\+=//'`
